test/fixtures/us_core_examples/Patient-ac1bdb14-fea1-4912-8d7c-e3ecec74b0d7.json

```diff
@@ -0,0 +1,139 @@
+{
+    "id":"ac1bdb14-fea1-4912-8d7c-e3ecec74b0d7",
+    "meta":{
+       "profile":[
+          "http://hl7.org/fhir/us/core/StructureDefinition/us-core-patient"
+       ]
+    },
+    "text":{
+    },
+    "extension":[],
```

Comment:
Are you expecting this to be an error or warning?

Reply:
Oh shoot, VS Code can be flakey with the vim plugin and not save things -- I had removed this to try it out with an empty element, changed it back, and the save didn't persist before I committed.  I reuploaded.


I was curious if this (or the other) would trigger `ele-1`, but it in playing around I realized that we are sanitizing the validation through fhir_models, so these would get stripped out anyhow.